pkg/pillar/types/zedroutertypes.go

```diff
@@ -1816,13 +1816,14 @@ type UnderlayNetworkStatus struct {
 	UnderlayNetworkConfig
 	ACLs int // drop ACLs field from UnderlayNetworkConfig
 	VifInfo
-	BridgeMac       net.HardwareAddr
-	BridgeIPAddr    string // The address for DNS/DHCP service in zedrouter
-	AllocatedIPAddr string // Assigned to domU
-	Assigned        bool   // Set to true once DHCP has assigned it to domU
-	IPAddrMisMatch  bool
-	HostName        string
-	ACLDependList   []ACLDepend
+	BridgeMac         net.HardwareAddr
+	BridgeIPAddr      string   // The address for DNS/DHCP service in zedrouter
+	AllocatedIPv4Addr string   // Assigned to domU
+	AllocatedIPv6List []string // IPv6 addresses assigned to domU
+	Assigned          bool     // Set to true once DHCP has assigned it to domU
```

Comment:
Maybe this could be renamed to `IPv4Assigned`?

Reply:
Yes, renamed the flag to IPv4Assigned. Thanks!